cpp/client/ct.cc

```diff
@@ -169,7 +169,7 @@ static LogVerifier* GetLogVerifierFromFlags() {
 static void AddOctetExtension(X509* cert, int nid, const unsigned char* data,
                               int data_len, int critical) {
   // The extension as a single octet string.
-  ScopedASN1_OCTET_STRING inner(ASN1_OCTET_STRING_new());
+  util::ScopedASN1_OCTET_STRING inner(ASN1_OCTET_STRING_new());
```

Comment:
You could put some `using` statements at the top (starting at line 127, keep them sorted), instead of changing it all over the file.